uefi-test-runner/src/proto/network.rs

```diff
@@ -0,0 +1,116 @@
+use uefi::{
```

Comment:
No substantive comment, just wanted to mention I appreciate the nice test here :)